src/planwise/model/import_job.clj

```diff
@@ -101,15 +101,16 @@
   (let [page (:page job)
         facility-ids (:facility-ids job)
         facility-count (count facility-ids)
-        [_ _ [_ page-ids total-pages sites-without-location]] event
+        [_ _ [_ page-ids total-pages sites-without-location sites-without-type]] event
```

Comment:
You also need to update the FSM definition, otherwise it won't import more than one page of sites from Resourcemap.

See https://github.com/instedd/planwise/commit/38fc76089601db1a93564ccd068c9dc56960476d for the last fix for a similar bug.

Also, it makes sense to re-group the return values into a hashmap. The length of that vector is beginning to get ridiculous.